src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -258,6 +275,26 @@ export class TerminalService implements ITerminalService {
 		this._refreshAvailableProfiles();
 	}
 
+	private _forwardInstanceHostEvents(host: ITerminalInstanceHost) {
+		host.onDidChangeInstances(this._onDidChangeInstances.fire, this._onDidChangeInstances);
+		host.onDidDisposeInstance(this._onDidDisposeInstance.fire, this._onDidDisposeInstance);
+		// Track the latest active terminal for each host so that when one becomes undefined (eg.
```

Comment:
not sure I understand what's going on here

Reply:
Apparently I stopped writing the comment mid sentence